tests/baselines/reference/user/enhanced-resolve.log

```diff
@@ -31,6 +90,16 @@ node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(235,4): error TS2322:
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(246,4): error TS2322: Type 'null' is not assignable to type '(path: any) => any'.
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(252,24): error TS2322: Type 'null' is not assignable to type '(path: any, callback: any) => void'.
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(257,28): error TS2322: Type 'null' is not assignable to type '(path: any) => any'.
+node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(261,21): error TS2339: Property 'provide' does not exist on type 'Storage'.
```

Comment:
Methinks there's a global type named `Storage` now which is overriding the jsdocy value-following fallback lookup.

Reply:
...no, that's not it. Something seems wrong.